### DISCUSSION TITLE
Does this rotate logs by default?

### DISCUSSION THREAD

**Discussion by asgeo1** [EasyEngine/easyengine#1598] [Support Requests]
I've just installed EE v4, and are a bit confused about logrotate.

I read this [help page on logrotate](https://easyengine.io/tutorials/linux/logrotate/) and it implies that logrotate works out-of-the-box. 

> You will need this if you are using **_custom_** location for log files.

Which to me means logrotate should be working if I haven't changed the default log location (which I haven't).  However, I'm not seeing anything which would run logrotate to cleanup nginx/php logs on either the host or inside container, so not sure it's setup or working.

It also mentions path `/var/www/example.com/logs` - which must be from an earlier version of EE, because the the current path appears to be `/opt/easyengine/sites/example.com/logs`

At this stage I just assume that help page is out-of-date with the current version of EE and logrotate probably needs to be manually setup.

I just want to make sure and that I'm not overlooking anything. Maybe that help page could get updated to make it less confusing.

**Comment by dhsathiya** [comment]
Yes, the document you shared is outdated. If you know how to configure log rotate then here are the configuration specific to EE4.
https://github.com/EasyEngine/feature-requests/issues/73#issuecomment-667002783
